jwst/pipeline/calwebb_image2.cfg

```diff
@@ -1,5 +1,7 @@
 name = "Image2Pipeline"
 class = "jwst.pipeline.Image2Pipeline"
+output_use_model = True
+suffix = "cal"
```

Comment:
Basically, this will work as in `calwebb_spec2` then. That would be the example for anything more complicated that just a step.